lib/utils.ts

```diff
@@ -50,3 +51,43 @@ export const toCompactNumber = (number: any) => {
   // Convert number to compact form 123,000,000 becomes 123M
   return new Intl.NumberFormat('en', { notation: 'compact', maximumSignificantDigits: 4 }).format(value);
 };
+
+// NOTE: Bringing these in from the updated OpenBook as we can't upgrade...
+// https://github.com/openbook-dex/openbook-v2/blob/d7d909c876e161d0a2bed9678c3dc5b9d0d430fb/ts/client/src/utils/utils.ts#L51
+// eslint-disable-next-line max-len
+export const toNative = (
+  uiAmount: number,
+  decimals: number
+) => new BN(Math.round(uiAmount * 10 ** decimals));
+
+// eslint-disable-next-line max-len
+export const toUiDecimals = (nativeAmount: number, decimals: number): number => nativeAmount / 10 ** decimals;
+
+export const priceUiToLots = (
```

Comment:
can it be simplified to have only two params like openbook ?
` uiPriceToLots(market.account, price)`

not so important but more readable

Reply:
It could, but this kept me honest for this version. Ideally we would just use openbook sdk, but there's breaking changes from v0.1.10 and v0.2.x so I manually added this and passed it on the side I could reference the values.